ibis/backends/dask/tests/test_schema.py

```diff
@@ -1,159 +1,14 @@
-                        90.899999999999991,
-                    ],
-                    dtype=np.float64,
-                ),
-                'float_col': np.array(
-                    [
-                        np.nan,
-                        1.1000000238418579,
-                        2.2000000476837158,
-                        3.2999999523162842,
-                        4.4000000953674316,
-                        5.5,
-                        6.5999999046325684,
-                        7.6999998092651367,
-                        8.8000001907348633,
-                        9.8999996185302734,
-                    ],
-                    dtype=np.float32,
-                ),
-                'int_col': np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], dtype='i4'),
-                'month': [11, 11, 11, 11, 2, 11, 11, 11, 11, 11],
-                'smallint_col': np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], dtype='i2'),
-                'string_col': [
-                    '0',
-                    '1',
-                    None,
-                    'double , whammy',
-                    '4',
-                    '5',
-                    '6',
-                    '7',
-                    '8',
-                    '9',
-                ],
-                'timestamp_col': [
-                    pd.Timestamp('2010-11-01 00:00:00'),
-                    None,
-                    pd.Timestamp('2010-11-01 00:02:00.100000'),
-                    pd.Timestamp('2010-11-01 00:03:00.300000'),
-                    pd.Timestamp('2010-11-01 00:04:00.600000'),
-                    pd.Timestamp('2010-11-01 00:05:00.100000'),
-                    pd.Timestamp('2010-11-01 00:06:00.150000'),
-                    pd.Timestamp('2010-11-01 00:07:00.210000'),
-                    pd.Timestamp('2010-11-01 00:08:00.280000'),
-                    pd.Timestamp('2010-11-01 00:09:00.360000'),
-                ],
-                'tinyint_col': np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], dtype='i1'),
-                'year': [
-                    2010,
-                    2010,
-                    2010,
-                    2010,
-                    2010,
-                    2009,
-                    2009,
-                    2009,
-                    2009,
-                    2009,
-                ],
-            }
-        ),
-        npartitions=npartitions,
-    )
-
-    expected = [
-        ('bigint_col', dt.int64),
-        ('bool_col', dt.boolean),
-        ('bool_obj_col', dt.boolean),
-        ('date_string_col', dt.string),
-        ('double_col', dt.float64),
-        ('float_col', dt.float32),
-        ('int_col', dt.int32),
-        ('month', dt.int64),
-        ('smallint_col', dt.int16),
-        ('string_col', dt.string),
-        ('timestamp_col', dt.timestamp),
-        ('tinyint_col', dt.int8),
-        ('year', dt.int64),
-    ]
-
-    assert sch.infer(df) == ibis.schema(expected)
-
-
+# TODO(kszucs): move it to ibis/formats
```

Comment:
Is this comment still valid?

Reply:
Yes, that tests the apply_to schema method which we still need to refactor in following patches.